qt/python/mantidqt/utility/__init__.py

```diff
@@ -0,0 +1,14 @@
+class SignalBlocker(object):
```

Comment:
Can we please add the GPL licence text (as the commented text in https://github.com/mantidproject/mantid/blob/master/qt/applications/workbench/workbench/__init__.py) to the head of each file?

This is a nice idea. Should we use Python's `contextlib` module to simplify it?

Also, can we move this to the `mantidqt.utils.qt` package?